File: Python/8py 3/other_angle.py
"""
|--------------------------------------------------------------------------
| Third Angle of a Triangle
|--------------------------------------------------------------------------
|
"""

def other_angle(a, b):
   return 180 - (a + b)


def other_angle(*a):
   return 180 - sum(a)


def other_angle(a,b):
   return 180 - a - b


print(other_angle(30, 60), 90)
print(other_angle(60, 60), 60)
print(other_angle(43, 78), 59)
print(other_angle(10, 20), 150)

File: PHP/8php 3/otherAngle.php
<?php
/*
|--------------------------------------------------------------------------
| Third Angle of a Triangle
|--------------------------------------------------------------------------
|
*/

function otherAngle($a, $b) {
   return 180 - ($a + $b);
}


function otherAngle($a, $b) {
   return 180 - $a - $b;
}


function otherAngle(...$n) {
   return 180 - array_sum($n);
}


print_r(otherAngle(30, 60) . ' ' . 90 . '
');
print_r(otherAngle(60, 60) . ' ' . 60 . '
');
print_r(otherAngle(43, 78) . ' ' . 59 . '
');
print_r(otherAngle(10, 20) . ' ' . 150 . '
');